handbook/company/communications.md

```diff
@@ -629,6 +629,29 @@ Fleet prioritizes a [bias for action](https://fleetdm.com/handbook/company#owner
 ## Receiving performance feedback
 At Fleet, performance feedback is a continuous process. We give feedback (particularly negative) as soon as possible. Managers at Fleet will provide performance feedback [during scheduled 1:1 meetings](https://fleetdm.com/handbook/company/leadership#performance-feedback), if not sooner.
 
+## Giving and receiving feedback
```

Comment:
- we use `-` not asterisks throughout the rest of the handbook
- indent bullet items to match others in handbook
- consolidate lists to match others in handbook (eliminate extra newlines)
- for "when giving feedback", use colon (same for other).  Leave off bold.  Eliminate extra whitespace.  Pattern match.
- consolidate this w/ previous section which is about receiving performance feedback

Reply:
Thanks for the feedback. Updated!